### DISCUSSION TITLE
The tab doesn't show up in WebUI : No such file or directory: './externals/Next_ViT/classification/nextvit.py'"

### DISCUSSION THREAD

**Discussion by BlinkerHigh** [thygate/stable-diffusion-webui-depthmap-script#63] [Q&A] [answered]
Hello, the tab won't show up after installing the extension.
Installing both from the extensions tab in WebUI or copy-pasting the GitHub URL have the same effect.
I even did a full reset of WebUI.
Any idea?

**Comment by thygate** [comment]
Do you have any error messages in the console ?

**Reply by BlinkerHigh** [reply]
This is the one I think.
"FileNotFoundError: [Errno 2] No such file or directory: './externals/Next_ViT/classification/nextvit.py'"

**Reply by BlinkerHigh** [reply]
I know someone else has the same error.

**Comment by thygate** [comment]
See issue https://github.com/thygate/stable-diffusion-webui-depthmap-script/issues/55

**Reply by BlinkerHigh** [reply]
I'm not tech savvy enough, I don't understand what I'm supposed to do :)
I just used an automatic install.

**Reply by thygate** [reply]
ok, easiest solution for you is to clone the midas repo with commit hash mentioned in that thread, as follows : 

delete `stable-diffusion-webui/repositories/midas` folder

and manually download this zip of the midas repo at that commit hash : https://github.com/isl-org/MiDaS/archive/66882994a432727317267145dc3c2e47ec78c38a.zip

extract the zip to  `stable-diffusion-webui/repositories/`

the extracted folder will be called `MiDaS-66882994a432727317267145dc3c2e47ec78c38a`, so rename it to `midas`

so you will again have a folder `stable-diffusion-webui/repositories/midas`

case sensitive

don't forget to restart the backend.

**Reply by BlinkerHigh** [reply]
Thank you so much!

**Comment by thygate** [comment, marked as answer]
Should be fixed in latest version 0.3.3
